fdbserver/GrvProxyTagThrottler.actor.cpp

```diff
@@ -438,3 +444,25 @@ TEST_CASE("/GrvProxyTagThrottler/Fifo") {
 	wait(mockFifoClient(&throttler));
 	return Void();
 }
+
+// Tests that while throughput is low, the tag throttler
+// does not accumulate too much budget.
+TEST_CASE("/GrvProxyTagThrottler/LimitedIdleBudget") {
+	state GrvProxyTagThrottler throttler(5.0);
+	state TagSet tagSet;
+	state TransactionTagMap<uint32_t> counters;
+	{
+		TransactionTagMap<double> rates;
+		rates["sampleTag"_sr] = 10.0;
+		throttler.updateRates(rates);
+	}
+	tagSet.addTag("sampleTag"_sr);
+
+	state Future<Void> server = mockServer(&throttler);
+	wait(delay(60.0));
+	state Future<Void> client = mockClient(&throttler, TransactionPriority::DEFAULT, tagSet, 1, 20.0, &counters);
+	wait(timeout(client && server, 60.0, Void()));
+	TraceEvent("TagQuotaTest_LimitedIdleBudget").detail("Counter", counters["sampleTag"_sr]);
+	ASSERT(isNear(counters["sampleTag"_sr], 60.0 * 10.0));
```

Comment:
Where are these numbers coming from? Is the `10` the rate set earlier? Is the `60` the delay used above? If so, could you clarify by using variable names; if not, could you add a comment?